### PR TITLE
feat: add --extraMountOptions parameter for libfuse, WinFsp and OSXFuse

### PR DESCRIPTION
**Pull request by arvgord**
[#21] With this option, a user can add additional mount options during the start of DICOMFuse to get the desired behavior

### CODE REVIEW

**Comment by dzelemba** [comment]
Can we add a unit test for the AppMountOptions file?

**Comment by arvgord** [comment]
I tested --extraMountOptions parameter manually for all operating systems and this works. List of options jcommander parses well. If I create a unit test in this class I'll get an integration test with external dependency from libfuse(WinFsp, OSXFuse) like this [PR](https://github.com/GoogleCloudPlatform/healthcare-api-dicom-fuse/pull/19). 
What part of this class do you want to check? If you want to check that all arguments are parsed, I can add [similar test](https://github.com/cbeust/jcommander).

**Comment by dzelemba** [comment]
Ya just a simple unit tests that mocks out the libfuse dependency should be sufficient here. I just want a test that exercises the logic in the AppMountProcess file (i.e. check the params passed to dicomfuse for each OS with and without extra mount options).

**Comment by arvgord** [comment]
Done

File: src/test/java/com/google/dicomwebfuse/AppMountProcessTest.java [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    DicomFuse dicomFuse = Mockito.mock(DicomFuse.class);
+    ArgumentCaptor<String[]> captor = ArgumentCaptor.forClass(String[].class);
+    Mockito.doNothing()
+        .when(dicomFuse)
+        .mount(any(), any(Boolean.class), any(Boolean.class), captor.capture());
+    // When
+    AppMountProcess appMountProcess = new AppMountProcess(arguments, os, parameters, dicomFuse);
+    appMountProcess.startMountProcess();
+    // Then
+    List<String> options = Arrays.asList(captor.getValue());
```
Comment (dzelemba) [inline src/test/java/com/google/dicomwebfuse/AppMountProcessTest.java]: assertThat(options, CoreMatchers.hasItems())

Comment (arvgord) [reply src/test/java/com/google/dicomwebfuse/AppMountProcessTest.java]: Done

**Pull request merged**